shared/charts/gpii-preferences/values.yaml

```diff
@@ -29,7 +29,7 @@ dnsNames:
 - preferences.test.local
 
 ingress:
-  disable_ssl_redirect: false
+  disableSslRedirect: false
```

Comment:
What was wrong with `disable_ssl_redirect`?

Reply:
https://github.com/gpii-ops/gpii-infra/pull/230#discussion_r241529042

Reply:
Got it, thanks!